ios/FluentUI/Command Bar/CommandBarButton.swift

```diff
@@ -65,6 +66,7 @@ class CommandBarButton: UIButton {
         titleLabel?.isEnabled = isEnabled
         titleLabel?.font = item.titleFont
         accessibilityLabel = (accessibilityDescription != nil) ? accessibilityDescription : title
+        accessibilityHint = item.accessibilityHint
```

Comment:
are you losing any default OS behavior by overriding the accessibilityHint always to nil

Reply:
Good point. Checked that UIButton doesn't have a default value for a11y hint. 
<img width="1046" alt="Screen Shot 2021-07-22 at 1 59 36 PM" src="https://user-images.githubusercontent.com/59622334/126709024-fdb39c52-85e7-4951-937d-1126c261da8c.png">